### ISSUE TITLE
<a class="tag" href="..."> has white text on white bg, when in primary hero

### ISSUE THREAD

**Issue by CapacitorSet** [jgthms/bulma#1216]
### Overview of the problem

This is about the Bulma **CSS framework**
I'm using Bulma **version** 0.5.3
My **browser** is: Firefox Nightly 57.0a1 (2017-09-20)
I am sure this issue is **not a duplicate**? Yes, only #770 is similar but not quite.

### Steps to Reproduce

Create a page with Bulma and the following code (a minimal testcase):
```html
<section class="hero is-primary">
	<a class="tag" href="https://github.com/">Hello!</a>
</section>
```

### Expected behavior

A tag containing "Hello!" in a small hero should be displayed.

### Actual behavior

The tag is displayed indeed, but as white text on a white background.

Note that this also happens in more complex examples, as long as the hero is primary. This is also fixed by using `<span class="tag"><a>` instead.

**Comment by connectkushal** [comment]
@jgthms is adding `.tag` to `a:not(.button, .tag)` in the hero.sass a correct solution for the issue ? if so i can add it to a fix (#1203) for a similar issue (#935) and also update the title of the pr.

**Comment by connectkushal** [comment]
This has been fixed in #1203 :) please close this issue.
